src/shared/utilities/isMobile.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * A utility to determine if a browser is mobile.
+ * @returns A boolean determining if the browser is a mobile version
+ */
+export const isMobile = (): boolean => {
```

Comment:
this looks like a maintenance nightmare. is it something we can/should install and update to latest on occasion?

Reply:
The closest is ua-parser, but these packages are security risks, doing it this way ensures we're code-safe.